repl.html

```diff
@@ -37,7 +37,7 @@
 Output:
 </div>
 <div>
-<code><span id="prompt-string">>>></span></code><input id="input" type="text" />
+<input id="input" type="text" /><input id="input-button" type="submit" value="Submit" onclick=submitInput />
```

Comment:
can you register the onClick in the JS please?

Reply:
Sure but what does that mean?

Reply:
`#input-button.addEventListener` etc like the others

Reply:
Done!